### PR TITLE
Remove activesupport dependency

### PR DESCRIPTION
**Pull request by rearden-steel**
Make Monero gem great again.

### CODE REVIEW

File: lib/rpc/wallet.rb [hunk 1]
```diff
@@ -97,7 +97,7 @@ def self.get_transfers(args={})
       h = Hash.new
       json = RPC::Client.request("get_transfers", options)
       json.map{|k, v|
-        h[k] = v.collect{|transfer| (RPC.config.transfer_clazz || "RPC::IncomingTransfer").constantize.new(transfer)}
+        h[k] = v.collect{|transfer| Module.const_get((RPC.config.transfer_clazz || "RPC::IncomingTransfer")).new(transfer) }
```
Comment (krtschmr) [inline lib/rpc/wallet.rb:100]: did you test that LoC?

**Review by krtschmr — APPROVED** [review]
should be good

File: lib/rpc/wallet.rb (hunk 1 above)
Comment (rearden-steel) [reply lib/rpc/wallet.rb:100]: Yes, sure, it works.

Comment (krtschmr) [reply lib/rpc/wallet.rb:100]: merged and pushed to rubygems already.

**Pull request merged**